server/service/src/sync/translations/requisition.rs

```diff
@@ -91,35 +93,47 @@ pub struct LegacyRequisitionRow {
 
     #[serde(rename = "lastModifiedAt")]
     pub last_modified_at: i64,
-    #[serde(deserialize_with = "empty_str_as_option")]
+    #[serde(deserialize_with = "empty_str_as_option_string")]
     pub requester_reference: Option<String>,
-    #[serde(deserialize_with = "empty_str_as_option")]
+    #[serde(deserialize_with = "empty_str_as_option_string")]
     pub linked_requisition_id: Option<String>,
     /// min_months_of_stock
     pub thresholdMOS: f64,
     /// relates to max_months_of_stock
     pub daysToSupply: i64,
 
-    #[serde(deserialize_with = "empty_str_as_option")]
+    #[serde(deserialize_with = "empty_str_as_option_string")]
     pub comment: Option<String>,
 
+    #[serde(default)]
     #[serde(rename = "om_created_datetime")]
+    #[serde(deserialize_with = "empty_str_as_option_generic")]
```

Comment:
`#[serde(default)]` Handles the key/value not being included at all. Needed after adding `#[serde(deserialize_with = "empty_str_as_option_generic")]` to handle value being `""`